metals/src/main/scala/scala/meta/internal/metals/codeactions/ImportMissingSymbol.scala

```diff
@@ -43,14 +44,42 @@ class ImportMissingSymbol(compilers: Compilers) extends CodeAction {
         }
     }
 
+    def addImportAllMissingSymbol(
+        codeActions: Buffer[l.CodeAction]
```

Comment:
You can probably just leave this as a `Seq` instead of a buffer. Down below at the end of the file where this is `flattened` and passed it, it can just stay a `Seq`. Then in this method, you could change the logic a bit that instead of the `if` statement adding something to the buffer, you could do something like


```scala
val codeActionsToReturn = If(uniqueCodeActions.length > 1) {
  ... // create new code action
  codeActions :+ newCodeAction
} else {
  codeActions
}
codeActionsToReturn
```